src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,21 @@
+%normal-text {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
```

Comment:
```suggestion
```
font-family is better to add to HTML selector
font-style: normal; id default value, there is no need in it